server/chat-commands/moderation.ts

```diff
@@ -806,13 +806,14 @@ export const commands: ChatCommands = {
 
 		// Use default time for locks.
 		const duration = week ? Date.now() + 7 * 24 * 60 * 60 * 1000 : (month ? Date.now() + 30 * 24 * 60 * 60 * 1000 : null);
+		let punishment;
 		let affected = [];
 
 		if (targetUser) {
 			const ignoreAlts = Punishments.sharedIps.has(targetUser.latestIp);
-			affected = await Punishments.lock(targetUser, duration, null, ignoreAlts, userReason);
+			({punishment, affected} = await Punishments.lock(targetUser, duration, null, ignoreAlts, userReason));
```

Comment:
```suggestion
			({punishment, affected}) = await Punishments.lock(targetUser, duration, null, ignoreAlts, userReason);
```

I would prefer this style for destructuring.

Reply:
Does that syntax work? Just tried it in the Node REPL and it didn't seem to work.

Reply:
Oh, sorry, my mistake.